internal/consumer/options.go

```diff
@@ -32,8 +34,9 @@ func NewOptions() *Options {
 		MaxPollInterval:    "300000",
 		EnableAutoCommit:   "false",
 		AutoOffsetReset:    "earliest",
-		StatisticsInterval: "60000",
+		StatisticsInterval: "30000",
```

Comment:
Not sure if I missed something, why change from 60000, is it too long?

Reply:
honestly, i think we'll need to figure out a sweet spot but i bumped it to 60,000 when i was testing retry loops and it was annoying when stats messages came in every 15 seconds because by the time i was done debugging, i had a bunch of stats messages. Long term 30 seconds seems reasonable to poll for stats and update metrics is what i was thinking? 60 seconds seemed long for when we get to  actively monitoring the client. we can always override it in config but wanted a "sane" default.